packages/rspeedy/core/src/cli/inspect.ts

```diff
@@ -28,6 +29,11 @@ export async function inspect(
       configPath: inspectOptions.config,
     })
 
+    if (inspectOptions.base) {
```

Comment:
I don't think we need `--base` for `rspeedy inspect` since there wouldn't be any server running.

Should we add it as a common option and use in `dev.ts` and `preview.ts`?

https://github.com/lynx-family/lynx-stack/blob/d7301017a383b8825cdc813a649ef26ce1c37641/packages/rspeedy/core/src/cli/commands.ts#L17-L23